pages/dashboard.py

```diff
@@ -5,7 +5,7 @@
 class DashboardPage(OSFBasePage):
 
     locator_dictionary = {
-        'create_project_button':(By.CSS_SELECTOR, 'button.btn-success:nth-child(1)'),
+        'create_project_button':(By.CSS_SELECTOR, 'button.btn-success:nth-child(1)', long_timeout),
```

Comment:
In the original form of the base page, @brianjgeiger mentioned that should be able to overwrite our wait times for specific elements if we needed to.

While every class can create their own wait time if they wish - I kind of like to believe that very few will and will just inherit `settings.TIMEOUT` that was set in `BaseElement`. 

I was uncertain how to name the variables these timeouts. What might make the sense is to have variables for short, medium, and long timeouts in settings and just call those when a specific element needs a longer wait time. Would that make this better?

Reply:
I see. I like the last idea of using `settings` then  overriding these for different environments say with `local` locally, `local-travis` for staging, `local-travis` for prod if that's a thing. Since, it would take some time to determine these, maybe not worry about it right now.